node/blockchain/inner_models/block_message/coin_transfer.py

```diff
@@ -41,6 +41,7 @@ def _make_recipients_account_states(cls, request, blockchain_facade) -> dict[Acc
             updated_amounts[recipient] += transaction.amount
 
         updated_account_states = {
-            account_number: AccountState(balance=amount) for account_number, amount in updated_amounts.items()
+            account_number: AccountState(balance=amount, account_lock=request.make_hash())
```

Comment:
@olegtropinin no, recipient account lock does not change when coin received. You must keep current account lock. Take it from blockchain facade

Reply:
I have updated facade method get_account_lock to handle case when account_state.account_lock is None